app/controllers/v0/mhv_opt_in_flags_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module V0
+  class MHVOptInFlagsController < ApplicationController
+    def show
+      opt_in_flag = MHVOptInFlag.find_by(user_account_id: current_user.account_uuid, feature: params[:feature])
+      raise Common::Exceptions::RecordNotFound, current_user.account_uuid if opt_in_flag.nil?
+
+      render json: { user: current_user.email, status: 'OK' }
+    end
+
+    def create
+      opt_in_flag = MHVOptInFlag.find_or_create_by(user_account_id: current_user.account_uuid,
+                                                   feature: params[:feature])
+      raise Common::Exceptions::ValidationErrors, opt_in_flag unless opt_in_flag.valid?
+
+      opt_in_flag.save
+      render json: { user: current_user.email, status: 'OK' }
```

Comment:
We should render the same thing as `show`, with just a slight difference with the `status`, if the create occurs successfully, so:

`render json: { mhv_opt_in_flag: { user_account_id: opt_in_flag.user_account.id, feature: opt_in_flag.feature }, status: :created }`